accounts.go

```diff
@@ -49,6 +49,15 @@ func (lw *LibWallet) GetAccountsRaw(requiredConfirmations int32) (*Accounts, err
 	}, nil
 }
 
+func (accounts *Accounts) Next() *Account {
+	if accounts.Iter.CurrentIndex < len(accounts.Acc) {
```

Comment:
Can find where `accounts.Iter` was initialized. Do this code run without errors?

Reply:
Yes, it runs. 
I use it like this:

`func walletAccounts(confirmations: Int32) -> [DcrlibwalletAccount] {
        var accounts = [DcrlibwalletAccount]()
        do {
            let accountsRaw = try self.getAccountsRaw(confirmations)
            accountsRaw.iter = DcrlibwalletAccountIterator()
            
            while let account = accountsRaw.next() {
                accounts.append(account)
            }
            
            return accounts
        } catch let error {
            print("Error fetching wallet accounts: \(error.localizedDescription)")
            return accounts
        }
}`

Reply:
Commited a nicer implementation: cb25842